proposals/csharp-12.0/experimental-attribute.md

```diff
@@ -0,0 +1,53 @@
+ExperimentalAttribute
+=====================
+Report warnings for references to types and members marked with `System.Diagnostics.CodeAnalysis.ExperimentalAttribute`.
+```
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Assembly |
+                    AttributeTargets.Module |
+                    AttributeTargets.Class |
+                    AttributeTargets.Struct |
+                    AttributeTargets.Enum |
+                    AttributeTargets.Constructor |
+                    AttributeTargets.Method |
+                    AttributeTargets.Property |
+                    AttributeTargets.Field |
+                    AttributeTargets.Event |
+                    AttributeTargets.Interface |
+                    AttributeTargets.Delegate, Inherited = false)]
+    public sealed class ExperimentalAttribute : Attribute
+    {
+        public ExperimentalAttribute(string diagnosticId)
+        {
+            DiagnosticId = diagnosticId;
+        }
+
+        public string DiagnosticId { get; }
+        public string? UrlFormat { get; set; }
+    }
+}
+```
+
+## Warnings
+The warning message is a specific message, where `'{0}'` is the fully-qualified type or member name.
+```
+'{0}' is for evaluation purposes only and is subject to change or removal in future updates.
+```
+
+The warning is reported for any reference to a type or member that is either:
+- marked with the attribute,
+- in an assembly or module marked with the attribute,
+except when the reference occurs  within `[Experimental]` members (automatic suppression).
```

Comment:
I assume the "except ..." applies to both, but without a line break here, it is rendered in the latter bullet point only.
```suggestion

except when the reference occurs within `[Experimental]` members (automatic suppression).
```